paasta_tools/deployd/workers.py

```diff
@@ -129,9 +130,10 @@ def process_service_instance(self, service_instance):
                               bounce_again_in_seconds,
                           ))
         else:
-            bounce_timers.bounce_length.stop()
             self.log.info("{}.{} in steady state".format(
                 service_instance.service,
                 service_instance.instance,
             ))
+        if service_instance.reprocessed_count > 0:
+            bounce_timers.bounce_length.stop()
```

Comment:
I don't quite get this. 

Maybe we should be measuring the time to run s_m_j *and* the total bounce time?

Reply:
We do measure the time for both already. The only thing I'm trying to change here is not measuring total bounce time for service instances that are only processed once by a worker to check that they are in the correct state. I want the averages of total bounce time to only include things that are really being bounced (not just being checked by deployd).

Reply:
Oh I do have an indentation error though I think...